_includes/post-preview.html

```diff
@@ -0,0 +1,31 @@
+{% assign post = include.post %}
+
+{%- capture excerpt_link_url -%}
+  {%- if post.excerpt_link_url -%}
+    {{ post.excerpt_link_url }}
+  {%- else -%}
+    {{ site.baseurl }}{{ post.url }}
+  {%- endif -%}
+{%- endcapture -%}
+
+<article id="{{ post.id | default: post.title | slugify }}" class="post-preview">
```

Comment:
**polish**: Maybe a stylistic choice, but moving filters and other operations apart seems more readable.

```suggestion
{% assign post_id = post.title | default: post.id | slugify %}
<article id="{{ post_id }}" class="post-preview">
```

Reply:
This feels cleaner. Updated in [c4d016a](https://github.com/uswds/uswds-site/pull/2979/commits/c4d016a972835cafeb014ae58c443dc913828ca4)